File: src/lib/pagine-legali-defaults.ts
export const DEFAULT_PRIVACY_POLICY = `<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">1. Titolare del trattamento</h2>
  <p>
    Il titolare del trattamento dei dati personali è <strong>[Nome organizzazione]</strong>, con sede legale in
    [indirizzo], C.F./P.IVA: [inserire], email:
    <a href="mailto:[email privacy]" class="text-red-600 hover:underline">[email privacy]</a>.
  </p>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">2. Tipologie di dati trattati</h2>
  <p>Trattiamo le seguenti categorie di dati personali:</p>
  <ul class="list-disc pl-5 mt-2 space-y-1">
    <li><strong>Dati di registrazione:</strong> nome, cognome, indirizzo email, numero di telefono, password (in forma hash irreversibile).</li>
    <li><strong>Dati di prenotazione:</strong> dati dei partecipanti ai corsi (nome, cognome, email, telefono).</li>
    <li><strong>Dati di pagamento:</strong> ricevuta del bonifico bancario caricata dall'utente (file immagine/PDF).</li>
    <li><strong>Dati di navigazione:</strong> indirizzo IP, tipo di browser, pagine visitate, durata della sessione (se consenso ai cookie analitici).</li>
    <li><strong>Dati di consenso:</strong> data e tipo di consenso espresso in sede di registrazione e/o gestione cookie.</li>
  </ul>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">3. Finalità e basi giuridiche del trattamento</h2>
  <div class="overflow-x-auto">
    <table class="w-full text-xs border-collapse border border-gray-200 rounded-lg overflow-hidden">
      <thead class="bg-gray-50">
        <tr>
          <th class="text-left px-3 py-2 border-b border-gray-200 font-medium text-gray-700">Finalità</th>
          <th class="text-left px-3 py-2 border-b border-gray-200 font-medium text-gray-700">Base giuridica</th>
        </tr>
      </thead>
      <tbody class="divide-y divide-gray-100">
        <tr>
          <td class="px-3 py-2">Gestione del profilo utente e autenticazione</td>
          <td class="px-3 py-2">Esecuzione del contratto (art. 6, c. 1, lett. b GDPR)</td>
        </tr>
        <tr class="bg-gray-50">
          <td class="px-3 py-2">Gestione prenotazioni e pagamenti</td>
          <td class="px-3 py-2">Esecuzione del contratto (art. 6, c. 1, lett. b GDPR)</td>
        </tr>
        <tr>
          <td class="px-3 py-2">Invio email transazionali (conferme, attestati, reset password)</td>
          <td class="px-3 py-2">Esecuzione del contratto (art. 6, c. 1, lett. b GDPR)</td>
        </tr>
        <tr class="bg-gray-50">
          <td class="px-3 py-2">Adempimenti fiscali e contabili</td>
          <td class="px-3 py-2">Obbligo legale (art. 6, c. 1, lett. c GDPR)</td>
        </tr>
        <tr>
          <td class="px-3 py-2">Analisi statistica anonima del sito</td>
          <td class="px-3 py-2">Consenso (art. 6, c. 1, lett. a GDPR)</td>
        </tr>
        <tr class="bg-gray-50">
          <td class="px-3 py-2">Comunicazioni commerciali e marketing</td>
          <td class="px-3 py-2">Consenso (art. 6, c. 1, lett. a GDPR)</td>
        </tr>
      </tbody>
    </table>
  </div>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">4. Modalità del trattamento e conservazione</h2>
  <p>
    I dati sono trattati con strumenti elettronici su server ubicati nell'Unione Europea. Adottiamo misure
    tecniche e organizzative adeguate (cifratura, controllo accessi, backup) per garantire la sicurezza dei dati.
  </p>
  <p class="mt-2">I dati sono conservati per i seguenti periodi:</p>
  <ul class="list-disc pl-5 mt-2 space-y-1">
    <li>Dati del profilo: per tutta la durata del rapporto contrattuale e fino a 2 anni dalla cancellazione dell'account.</li>
    <li>Dati di prenotazione e pagamento: 10 anni per obblighi fiscali.</li>
    <li>Log di sistema: 12 mesi.</li>
    <li>Dati di marketing: fino a revoca del consenso.</li>
  </ul>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">5. Destinatari dei dati</h2>
  <p>I dati possono essere comunicati a:</p>
  <ul class="list-disc pl-5 mt-2 space-y-1">
    <li>Fornitori di servizi cloud e hosting (responsabili del trattamento ex art. 28 GDPR).</li>
    <li>Fornitori di servizi email transazionali.</li>
    <li>Autorità pubbliche, su richiesta e nei limiti di legge.</li>
  </ul>
  <p class="mt-2">I dati non vengono ceduti a terzi per scopi propri né trasferiti fuori dallo Spazio Economico Europeo.</p>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">6. Diritti degli interessati</h2>
  <p>Ai sensi degli artt. 15-22 del GDPR, hai il diritto di:</p>
  <ul class="list-disc pl-5 mt-2 space-y-1">
    <li><strong>Accesso</strong> (art. 15): ottenere conferma e copia dei tuoi dati.</li>
    <li><strong>Rettifica</strong> (art. 16): correggere dati inesatti o incompleti.</li>
    <li><strong>Cancellazione</strong> (art. 17): richiedere l'eliminazione dei dati («diritto all'oblio»).</li>
    <li><strong>Limitazione</strong> (art. 18): limitare il trattamento in determinati casi.</li>
    <li><strong>Portabilità</strong> (art. 20): ricevere i tuoi dati in formato strutturato.</li>
    <li><strong>Opposizione</strong> (art. 21): opporti al trattamento per marketing o per legittimo interesse.</li>
    <li><strong>Revoca del consenso</strong>: in qualsiasi momento, senza pregiudizio per la liceità del trattamento anteriore.</li>
  </ul>
  <p class="mt-2">
    Puoi esercitare i tuoi diritti dalla tua area personale oppure scrivendo a
    <a href="mailto:[email privacy]" class="text-red-600 hover:underline">[email privacy]</a>.
    Risponderemo entro 30 giorni.
  </p>
  <p class="mt-2">
    Hai inoltre il diritto di proporre reclamo all'autorità di controllo italiana:
    <strong>Garante per la Protezione dei Dati Personali</strong>, Piazza Venezia 11, Roma —
    <a href="https://www.garanteprivacy.it" class="text-red-600 hover:underline" target="_blank" rel="noopener noreferrer">www.garanteprivacy.it</a>.
  </p>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">7. Cookie</h2>
  <p>
    Per informazioni dettagliate sui cookie utilizzati consulta la
    <a href="/cookie-policy" class="text-red-600 hover:underline">Cookie Policy</a>.
  </p>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">8. Modifiche alla Privacy Policy</h2>
  <p>
    Ci riserviamo il diritto di aggiornare la presente informativa. In caso di modifiche sostanziali
    ti notificheremo via email. La versione più recente è sempre disponibile a questo indirizzo.
  </p>
</section>`;

export const DEFAULT_COOKIE_POLICY = `<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">1. Cosa sono i cookie</h2>
  <p>
    I cookie sono piccoli file di testo che vengono memorizzati nel tuo browser quando visiti un sito web.
    Consentono al sito di ricordare le tue azioni e preferenze (come lingua, dimensioni dei caratteri e altre
    impostazioni di visualizzazione) per un determinato periodo di tempo, in modo da non doverle inserire
    ogni volta che torni sul sito.
  </p>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">2. Cookie utilizzati da questo sito</h2>

  <h3 class="font-semibold text-gray-800 mt-4 mb-2">2.1 Cookie strettamente necessari</h3>
  <p class="mb-2 text-xs text-green-700 bg-green-50 border border-green-200 rounded-lg px-3 py-1.5 inline-block">
    Sempre attivi — non richiedono consenso
  </p>
  <div class="overflow-x-auto mt-2">
    <table class="w-full text-xs border-collapse border border-gray-200 rounded-lg overflow-hidden">
      <thead class="bg-gray-50">
        <tr>
          <th class="text-left px-3 py-2 border-b border-gray-200 font-medium">Nome</th>
          <th class="text-left px-3 py-2 border-b border-gray-200 font-medium">Fornitore</th>
          <th class="text-left px-3 py-2 border-b border-gray-200 font-medium">Scopo</th>
          <th class="text-left px-3 py-2 border-b border-gray-200 font-medium">Scadenza</th>
        </tr>
      </thead>
      <tbody class="divide-y divide-gray-100">
        <tr>
          <td class="px-3 py-2 font-mono">next-auth.session-token</td>
          <td class="px-3 py-2">[Nome organizzazione]</td>
          <td class="px-3 py-2">Mantiene la sessione autenticata dell'utente</td>
          <td class="px-3 py-2">Sessione / 30 giorni</td>
        </tr>
        <tr class="bg-gray-50">
          <td class="px-3 py-2 font-mono">next-auth.csrf-token</td>
          <td class="px-3 py-2">[Nome organizzazione]</td>
          <td class="px-3 py-2">Protezione contro attacchi CSRF</td>
          <td class="px-3 py-2">Sessione</td>
        </tr>
        <tr>
          <td class="px-3 py-2 font-mono">cookie_consenso</td>
          <td class="px-3 py-2">[Nome organizzazione]</td>
          <td class="px-3 py-2">Memorizza le preferenze cookie dell'utente (localStorage)</td>
          <td class="px-3 py-2">1 anno</td>
        </tr>
      </tbody>
    </table>
  </div>

  <h3 class="font-semibold text-gray-800 mt-6 mb-2">2.2 Cookie analitici</h3>
  <p class="mb-2 text-xs text-amber-700 bg-amber-50 border border-amber-200 rounded-lg px-3 py-1.5 inline-block">
    Richiedono consenso
  </p>
  <p class="mt-2">
    Al momento non utilizziamo cookie analitici di terze parti. In caso di futura attivazione (es. Google
    Analytics in modalità anonimizzata) questa sezione verrà aggiornata prima dell'installazione.
  </p>

  <h3 class="font-semibold text-gray-800 mt-6 mb-2">2.3 Cookie di marketing e profilazione</h3>
  <p class="mb-2 text-xs text-red-700 bg-red-50 border border-red-200 rounded-lg px-3 py-1.5 inline-block">
    Richiedono consenso esplicito
  </p>
  <p class="mt-2">Al momento non utilizziamo cookie di marketing o profilazione di terze parti.</p>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">3. Come gestire i cookie</h2>

  <h3 class="font-semibold text-gray-800 mb-2">3.1 Tramite il banner del sito</h3>
  <p>
    Al primo accesso puoi scegliere quali categorie di cookie accettare. Puoi modificare le tue preferenze
    in qualsiasi momento cancellando il dato <code class="bg-gray-100 px-1 rounded text-xs">cookie_consenso</code>
    dal localStorage del tuo browser (DevTools → Application → Local Storage).
  </p>

  <h3 class="font-semibold text-gray-800 mt-4 mb-2">3.2 Tramite le impostazioni del browser</h3>
  <p>Puoi bloccare o eliminare i cookie dalle impostazioni del tuo browser:</p>
  <ul class="list-disc pl-5 mt-2 space-y-1">
    <li><strong>Chrome:</strong> Impostazioni → Privacy e sicurezza → Cookie e altri dati dei siti</li>
    <li><strong>Firefox:</strong> Opzioni → Privacy e sicurezza → Cookie e dati dei siti web</li>
    <li><strong>Safari:</strong> Preferenze → Privacy → Gestisci i dati dei siti web</li>
    <li><strong>Edge:</strong> Impostazioni → Cookie e autorizzazioni del sito</li>
  </ul>
  <p class="mt-2">Attenzione: disabilitare i cookie strettamente necessari potrebbe compromettere il funzionamento del sito.</p>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">4. Riferimenti normativi</h2>
  <p>
    La presente Cookie Policy è redatta in conformità al Regolamento UE 2016/679 (GDPR),
    al D.Lgs. 196/2003 (Codice Privacy) come modificato dal D.Lgs. 101/2018,
    e alle Linee guida del Garante per la protezione dei dati personali in materia di cookie
    (10 giugno 2021, doc. web n. 9677876).
  </p>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">5. Contatti</h2>
  <p>
    Per qualsiasi domanda relativa ai cookie scrivi a
    <a href="mailto:[email privacy]" class="text-red-600 hover:underline">[email privacy]</a>.
    Per maggiori informazioni sul trattamento dei dati personali consulta la
    <a href="/privacy-policy" class="text-red-600 hover:underline">Privacy Policy</a>.
  </p>
</section>`;

export const DEFAULT_TERMINI_CONDIZIONI = `<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">1. Accettazione dei termini</h2>
  <p>
    Utilizzando questa piattaforma, accetti integralmente i presenti Termini e Condizioni d'Uso.
    Se non accetti, ti invitiamo a non utilizzare la piattaforma.
  </p>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">2. Descrizione del servizio</h2>
  <p>La piattaforma consente agli utenti di:</p>
  <ul class="list-disc pl-5 mt-2 space-y-1">
    <li>Visualizzare il catalogo dei corsi formativi disponibili.</li>
    <li>Prenotare uno o più posti per sé o per terzi.</li>
    <li>Caricare la ricevuta del bonifico bancario a conferma del pagamento.</li>
    <li>Scaricare l'attestato di partecipazione dopo la conferma della segreteria.</li>
  </ul>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">3. Registrazione e account</h2>
  <p>
    Per effettuare una prenotazione è necessario registrarsi con un indirizzo email valido. L'utente è
    responsabile della riservatezza delle proprie credenziali di accesso e di tutte le attività svolte con
    il proprio account. Ci riserviamo il diritto di sospendere o disattivare account che violino i presenti termini.
  </p>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">4. Prenotazione e pagamento</h2>
  <ol class="list-decimal pl-5 space-y-2">
    <li>Al momento della prenotazione, il posto viene bloccato temporaneamente per il tempo indicato nel corso.</li>
    <li>L'utente deve effettuare il bonifico bancario alle coordinate indicate e caricare la ricevuta entro il termine stabilito.</li>
    <li>La prenotazione diventa definitiva solo dopo la verifica della ricevuta da parte della segreteria (<strong>stato: CONFERMATA</strong>).</li>
    <li>Se la ricevuta non viene caricata entro il timeout, il posto viene automaticamente liberato e la prenotazione passa allo stato <strong>SCADUTA</strong>.</li>
    <li>La segreteria può rifiutare una ricevuta non corrispondente al pagamento dovuto, comunicandone il motivo all'utente.</li>
  </ol>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">5. Disdette e rimborsi</h2>
  <p>Le condizioni di disdetta e rimborso sono specificate per ogni singolo corso. In assenza di indicazioni specifiche, si applicano le seguenti regole:</p>
  <ul class="list-disc pl-5 mt-2 space-y-1">
    <li>Disdetta oltre 7 giorni prima dell'inizio: rimborso integrale.</li>
    <li>Disdetta tra 3 e 7 giorni prima dell'inizio: rimborso del 50%.</li>
    <li>Disdetta meno di 3 giorni prima: nessun rimborso.</li>
  </ul>
  <p class="mt-2">
    Per richiedere una disdetta contatta la segreteria all'indirizzo email indicato nel corso.
  </p>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">6. Attestati</h2>
  <p>
    Gli attestati di partecipazione vengono emessi dalla segreteria dopo la conferma del pagamento e
    al termine del corso. L'attestato è disponibile per il download nell'area personale dell'utente.
    L'attestato certifica la partecipazione al corso e non costituisce titolo professionale abilitante,
    salvo diversa indicazione nel corso stesso.
  </p>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">7. Limitazione di responsabilità</h2>
  <p>Il gestore della piattaforma non è responsabile per:</p>
  <ul class="list-disc pl-5 mt-2 space-y-1">
    <li>Interruzioni del servizio per cause di forza maggiore o manutenzione programmata.</li>
    <li>Errori nelle informazioni fornite dagli utenti in fase di prenotazione.</li>
    <li>Danni derivanti dall'uso improprio della piattaforma.</li>
  </ul>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">8. Proprietà intellettuale</h2>
  <p>
    Tutti i contenuti della piattaforma (testi, immagini, loghi, software) sono di proprietà del gestore
    o dei rispettivi titolari e sono protetti dalla normativa sul diritto d'autore.
    È vietata la riproduzione senza autorizzazione scritta.
  </p>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">9. Privacy e trattamento dei dati</h2>
  <p>
    Il trattamento dei dati personali è regolato dalla
    <a href="/privacy-policy" class="text-red-600 hover:underline">Privacy Policy</a>
    e dalla
    <a href="/cookie-policy" class="text-red-600 hover:underline">Cookie Policy</a>,
    che costituiscono parte integrante dei presenti Termini.
  </p>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">10. Legge applicabile e foro competente</h2>
  <p>
    I presenti Termini sono regolati dalla legge italiana. Per qualsiasi controversia è competente
    il Foro di [città], salvo diversa previsione di legge per i consumatori.
  </p>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">11. Modifiche ai termini</h2>
  <p>
    Ci riserviamo il diritto di modificare i presenti Termini in qualsiasi momento. Le modifiche
    sostanziali saranno comunicate via email agli utenti registrati con almeno 15 giorni di preavviso.
    L'utilizzo continuato della piattaforma dopo la comunicazione costituisce accettazione delle modifiche.
  </p>
</section>

<section>
  <h2 class="text-lg font-semibold text-gray-900 mb-3">12. Contatti</h2>
  <p>
    Per qualsiasi domanda relativa ai presenti Termini scrivi a
    <a href="mailto:[email info]" class="text-red-600 hover:underline">[email info]</a>.
  </p>
</section>`;

File: python_app/app/pagine_legali_defaults.py
DEFAULT_PRIVACY_POLICY = """
<h2>Informativa sulla Privacy</h2>
<p><em>Ultimo aggiornamento: {{DATA_AGGIORNAMENTO}}</em></p>

<h3>1. Titolare del trattamento</h3>
<p><strong>{{RAGIONE_SOCIALE}}</strong><br>
{{INDIRIZZO_SEDE}}<br>
Email: <a href="mailto:{{PRIVACY_EMAIL}}">{{PRIVACY_EMAIL}}</a></p>
<p>Per qualsiasi richiesta relativa ai dati personali scrivere all'indirizzo email sopra indicato.</p>

<h3>2. Dati raccolti</h3>
<p>Raccogliamo i seguenti dati personali:</p>
<ul>
  <li>Dati anagrafici (nome, cognome) forniti in fase di registrazione o prenotazione</li>
  <li>Indirizzo email utilizzato come credenziale di accesso e per le comunicazioni</li>
  <li>Numero di telefono e codice fiscale (opzionali, forniti in fase di prenotazione)</li>
  <li>Dati dei partecipanti ai corsi (nome, cognome, email, codice fiscale)</li>
  <li>Dati di pagamento (trattati esclusivamente dal provider di pagamento; non vengono memorizzati sul nostro server)</li>
  <li>Dati di navigazione (log tecnici del server web)</li>
  <li>Preferenze di comunicazione (aree di interesse per le notifiche sui corsi)</li>
</ul>

<h3>3. Finalità del trattamento</h3>
<p>I dati raccolti vengono utilizzati per:</p>
<ul>
  <li>Gestire l'account utente e le prenotazioni ai corsi (base giuridica: esecuzione del contratto)</li>
  <li>Inviare comunicazioni relative ai corsi prenotati: conferme, promemoria, attestati (base giuridica: esecuzione del contratto)</li>
  <li>Adempiere agli obblighi di legge fiscali e contabili (base giuridica: obbligo legale)</li>
  <li>Inviare comunicazioni di marketing sui prossimi corsi, <em>solo previo consenso esplicito</em> (base giuridica: consenso)</li>
</ul>

<h3>4. Età minima</h3>
<p>La piattaforma è destinata a persone di età pari o superiore a 16 anni. Non raccogliamo consapevolmente dati di minori di 16 anni. Se ritieni che un minore abbia fornito dati senza il consenso del genitore, contattaci all'indirizzo indicato.</p>

<h3>5. Conservazione dei dati</h3>
<ul>
  <li><strong>Dati account e prenotazioni:</strong> 10 anni dalla cessazione del rapporto contrattuale, salvo diversi obblighi di legge</li>
  <li><strong>Dati di marketing (lead non registrati):</strong> fino alla revoca del consenso o a 2 anni dall'ultima interazione</li>
  <li><strong>Log tecnici:</strong> 90 giorni</li>
  <li><strong>Token temporanei (reset password, verifica email):</strong> 1–7 giorni</li>
</ul>

<h3>6. Destinatari dei dati</h3>
<p>I dati non vengono ceduti a terzi per finalità commerciali. Possono essere comunicati a:</p>
<ul>
  <li>Provider di servizi email (per l'invio delle comunicazioni)</li>
  <li>Provider di pagamento Stripe / PayPal (per la gestione delle transazioni; soggetti a proprie informative)</li>
  <li>Cloudflare (CAPTCHA antibot, se abilitato; soggetto a propria informativa)</li>
  <li>Autorità competenti se richiesto dalla legge</li>
</ul>

<h3>7. Diritti dell'interessato</h3>
<p>In conformità al Regolamento UE 2016/679 (GDPR), l'interessato ha diritto di:</p>
<ul>
  <li><strong>Accesso</strong> — ottenere conferma che siano trattati dati che lo riguardano e riceverne copia</li>
  <li><strong>Rettifica</strong> — correggere dati inesatti o incompleti</li>
  <li><strong>Cancellazione</strong> ("diritto all'oblio") — richiedere la cancellazione dei propri dati</li>
  <li><strong>Opposizione</strong> — opporsi al trattamento, in particolare per finalità di marketing</li>
  <li><strong>Portabilità</strong> — ricevere i propri dati in formato strutturato e leggibile da macchina</li>
  <li><strong>Limitazione</strong> — chiedere la sospensione temporanea del trattamento</li>
  <li><strong>Reclamo</strong> — proporre reclamo al Garante per la Protezione dei Dati Personali (<a href="https://www.garanteprivacy.it" target="_blank">garanteprivacy.it</a>)</li>
</ul>
<p>Per esercitare i propri diritti è sufficiente inviare una richiesta a: <a href="mailto:{{PRIVACY_EMAIL}}">{{PRIVACY_EMAIL}}</a>. Risponderemo entro 30 giorni.</p>
<p>Gli utenti registrati possono inoltre:</p>
<ul>
  <li>Modificare i propri dati personali dalla sezione "Dati personali" del portale</li>
  <li>Scaricare una copia dei propri dati in formato JSON dalla sezione "Dati personali"</li>
  <li>Eliminare il proprio account dalla sezione "Dati personali"</li>
  <li>Revocare il consenso al marketing dalla sezione "Privacy e comunicazioni"</li>
</ul>
"""

DEFAULT_COOKIE_POLICY = """
<h2>Cookie Policy</h2>
<p><em>Ultimo aggiornamento: aggiornare con la data effettiva</em></p>

<h3>Cosa sono i cookie</h3>
<p>I cookie sono piccoli file di testo che i siti visitati inviano al browser dell'utente, dove vengono memorizzati per essere poi ritrasmessi agli stessi siti alla visita successiva.</p>

<h3>Cookie tecnici</h3>
<p>Questo sito utilizza esclusivamente cookie tecnici, necessari al funzionamento della piattaforma:</p>
{{TABELLA_COOKIE}}

<h3>Cookie di terze parti</h3>
<p>Alcune funzionalità opzionali possono caricare script di terze parti (es. gateway di pagamento, CAPTCHA). In tal caso vengono visualizzate specifiche informative prima del caricamento.</p>

<h3>Come disabilitare i cookie</h3>
<p>È possibile disabilitare i cookie nelle impostazioni del proprio browser. Si noti che la disabilitazione dei cookie tecnici può compromettere il funzionamento del sito.</p>
"""


def genera_tabella_cookie(app_name: str = "") -> str:
    _s_hdr = "text-align:left;padding:8px 12px;border:1px solid #e5e7eb;background:#f9fafb;font-weight:600"
    _s_td = "padding:8px 12px;border:1px solid #e5e7eb"
    _s_code = "background:#f3f4f6;padding:2px 6px;border-radius:4px;font-family:monospace"
    rows = [
        ("session", "Sessione autenticata dell'utente", "8 ore"),
        ("csrf_token", "Protezione CSRF dei moduli", "Sessione"),
        ("cookie_consenso", "Memorizza il consenso ai cookie", "1 anno"),
    ]
    tbody_rows = "".join(
        f'<tr>'
        f'<td style="{_s_td}"><span style="{_s_code}">{name}</span></td>'
        f'<td style="{_s_td}">{scopo}</td>'
        f'<td style="{_s_td}">{durata}</td>'
        f'</tr>'
        for name, scopo, durata in rows
    )
    return (
        f'<table style="width:100%;border-collapse:collapse;font-size:0.9em">'
        f'<thead><tr>'
        f'<th style="{_s_hdr}">Cookie</th>'
        f'<th style="{_s_hdr}">Scopo</th>'
        f'<th style="{_s_hdr}">Durata</th>'
        f'</tr></thead>'
        f'<tbody>{tbody_rows}</tbody>'
        f'</table>'
    )

DEFAULT_TERMINI = """
<h2>Termini e Condizioni di Utilizzo</h2>
<p><em>Ultimo aggiornamento: aggiornare con la data effettiva</em></p>

<h3>1. Accettazione dei termini</h3>
<p>L'utilizzo della piattaforma e la prenotazione dei corsi implicano l'accettazione integrale dei presenti termini e condizioni. Si prega di leggerli attentamente prima di procedere alla registrazione.</p>

<h3>2. Registrazione e account</h3>
<p>Per prenotare un corso è necessario creare un account fornendo dati veritieri e aggiornati. L'utente è responsabile della custodia delle proprie credenziali di accesso e di tutte le attività effettuate tramite il proprio account.</p>

<h3>3. Prenotazione e pagamento</h3>
<ul>
  <li>La prenotazione è confermata solo a seguito del pagamento integrale della quota di partecipazione (o immediatamente per i corsi gratuiti)</li>
  <li>Il pagamento può avvenire tramite i metodi indicati sulla piattaforma (bonifico bancario, carta di credito, PayPal)</li>
  <li>In caso di pagamento tramite bonifico, il posto è riservato per il periodo indicato; in assenza di conferma entro tale termine, la prenotazione viene annullata automaticamente</li>
</ul>

<h3>4. Cancellazioni e rimborsi</h3>
<p>Le politiche di cancellazione e rimborso sono definite per ciascun corso. Salvo diversa indicazione:</p>
<ul>
  <li>Cancellazione fino a 7 giorni prima del corso: rimborso integrale</li>
  <li>Cancellazione da 7 a 3 giorni prima: rimborso del 50%</li>
  <li>Cancellazione negli ultimi 3 giorni: nessun rimborso</li>
</ul>
<p>L'ente si riserva il diritto di annullare un corso per cause di forza maggiore, con rimborso integrale ai partecipanti iscritti.</p>

<h3>5. Attestati di partecipazione</h3>
<p>Al termine dei corsi per cui è previsto, viene rilasciato un attestato di partecipazione ai partecipanti che hanno completato il percorso formativo. L'attestato viene inviato per email o reso disponibile per il download dalla propria area personale.</p>

<h3>6. Proprietà intellettuale</h3>
<p>Tutti i materiali didattici, le presentazioni e i contenuti dei corsi sono di proprietà esclusiva dell'ente erogatore. È vietata la riproduzione, distribuzione o utilizzo a fini commerciali senza autorizzazione scritta.</p>

<h3>7. Limitazione di responsabilità</h3>
<p>L'ente non è responsabile per danni indiretti o consequenziali derivanti dall'utilizzo della piattaforma o dalla partecipazione ai corsi. La responsabilità massima è limitata all'importo pagato per il corso.</p>

<h3>8. Modifiche ai termini</h3>
<p>L'ente si riserva il diritto di modificare i presenti termini in qualsiasi momento. Le modifiche saranno comunicate agli utenti registrati via email con almeno 15 giorni di preavviso.</p>

<h3>9. Legge applicabile</h3>
<p>I presenti termini sono regolati dalla legge italiana. Per qualsiasi controversia è competente il Foro del luogo in cui ha sede l'ente erogatore.</p>
"""
